frontend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve ",
+    "start": "ng serve --host 0.0.0.0",
```

Comment:
maybe remove host, also check out baseUrl, or at least restrict URL to local rather than globally allow every address.